### ISSUE TITLE
Problem with graphite and Django 1.6

### ISSUE THREAD

**Issue by adrianlzt** [echocat/puppet-graphite#68]
Django 1.6 fails with Graphite as reported in https://github.com/graphite-project/graphite-web/commit/fc3f018544c19b90cc63797d18970a4cc27ef2ad#diff-e383725a971fca0685db19bfe7c65b32

I have make a workaround in the node manifest:
```
  # Fix graphite for Django 1.6
  if $::osfamily == "Debian" {
    exec { "fix_graphite_django1.6":
      command => '/usr/bin/find /opt/graphite/webapp/graphite -iname "urls.py" -exec /bin/sed -i s/"from django.conf.urls.defaults import \*"/"from django.conf.urls import \*"/ {} \;',
      onlyif => "/bin/grep -r 'from django.conf.urls.defaults import' /opt/graphite/webapp/graphite",
      require => Class['graphite'],
    }
  }
```

**Comment by tomharvey** [comment]
Do we expect v6 to reach puppet release soon or should we patch v5.10.2 while waiting?

**Comment by dwerder** [comment]
Better patch v5. v6 has some new conecpts, but I want to migrate smoothly. So I prefer small patches for v5.

**Comment by jstangroome** [comment]
If my PR #202 is accepted, v5 will be able to support Graphite-Web 0.9.12 on Ubuntu Trusty via:

```puppet
class { 'graphite':
  gr_django_pkg      => 'django',
  gr_django_ver      => '1.5',
  gr_django_provider => 'pip',
}

package { 'python-django':
  ensure => absent
}
```

**Comment by dwerder** [comment]
Fixed by #202
